quadcopter_brain/src/quadcopter_brain/quadcopter_brain.py

```diff
@@ -45,6 +45,24 @@ def hover_in_place(self):
         self.go_to_waypoints(waypoint_data)
         print("Hover command sent")
 
+    def go_to_faux_relative_waypoint(self, dEast, dNorth, dAlt=0,
+                                     time_to_sleep=15):
+        '''
+        Given a displacement in meters, this function calculates the desired
+        waypoint and tells the quadcopter to go there
+        '''
+        assert(isinstance(dEast, (int, float, long)))
```

Comment:
this seems unnecessary. I don't see any instance when this would be an issue when there wouldn't be other much larger issues